packages/app/MIGRATION.md

```diff
@@ -1 +1,62 @@
 # Migrations
+
+## x.y.z (DD.MM.YYYY)
+
+### Detailed Changelog
```

Comment:
I'd suggest to remove that line to be consistent with the structure of the file. We could just list all necessary information just under the version and below - as you did - the paths with their changes. What do you think?

Reply:
Sounds grät!